dashboard/app/views/pd/application/teacher_application_mailer/admin_approval.html.haml

```diff
@@ -41,7 +43,13 @@
   and refer to this
   = link_to('brochure', 'https://code.org/files/programs/codeorg-program-brochure.pdf')
   that highlights how to bring computer science to your school.
-  Please send any additional questions to your regional partner below.
+  - if @application.regional_partner
+    As
+    = @application.regional_partner.name
+    is Code.org's Regional Partner in your area, I am here to answer additional questions you
```

Comment:
I'm having a hard time understanding this sentence. If I fill in a name, I read it as "As Jeff is Code.org's Regional Partner in your area, I am here to answer additional questions you may have about the program or application." I'm not sure how the 'I' fits in with the 'Jeff'

Reply:
Sam had a [similar comment](https://docs.google.com/document/d/1HX7n1stbdmzzWrUi8--Z-6GuCRUqyzoNkZbH9w7TO7I/edit?disco=AAAA6MzsLtg) in the spec. @dmcavoy -- any thoughts here?

Reply:
regional_partner.name would give the name of an organization. Regional Partners have Program Managers who are people. Regional partners are organizations. So the sentence would read "As Educate Maine is Code.org's Regional Partner in your area...". How about?

"I am also here to answer additional questions you may have about the program or application. Thank you very much for your support of computer science for all!"

I don't think we need to establish the connection between RP and Code.org again here.

Reply:
sounds good! I'll update